src/Forwarding_DLL.h

```diff
@@ -1,2 +1,12 @@
-extern "C"  __declspec(dllexport) double* Forwarding(int lx, int ly, int point_count, int prism_count, double x_min, double dx, double x_max,
+#pragma once
+
+#ifdef _MSC_VER
+	#define DLLEXPORT __declspec(dllexport)
+#else
+	#define DLLEXPORT
+#endif
+
+extern "C" {
+	DLLEXPORT double* Forwarding(int lx, int ly, int point_count, int prism_count, double x_min, double dx, double x_max,

```

Comment:
The revised could not be built successfully, so we edited here, putting "DLLEXPORT" first.

Reply:
lgtm (looks good to me)